ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/node-type-l3-port-channels.yml

```diff
@@ -0,0 +1,93 @@
+---
+type: spine
+spine:
+  nodes:
+    - name: node-type-l3-port-channels
+      id: 1
+      evpn_role: none
+      loopback_ipv4_pool: 2.3.4.5/24
+      bgp_as: 65005
+      l3_port_channels:
+        - # Port-Channel with 3 member ports
+          name: Port-Channel2
+          mode: active
+          member_interfaces:
+            - name: Ethernet1
+              peer: peerDevice1
+              peer_interface: Ethernet11
+              speed: "forced 10000full"
+            - # peer, peer_interface not set, use one from parent L3 Port-Channel
+              name: Ethernet2
+              speed: "forced 10000full"
+            - # one with structured config for member port (TO ADD)
+              name: Ethernet3
+              description: "Custom eth3 description"
+          ip_address: 192.168.1.102/31
+          peer: peer1
+          peer_port_channel: Port-Channel10
+          peer_ip: 192.168.1.10
+          static_routes:
+            - prefix: 0.0.0.0/0
+          qos_profile: TEST-QOS-PROFILE1
+          wan_carrier: Cybercast
+          wan_circuit_id: 101
+          flow_tracking:
+            enabled: true
+        - # Port-Channel with 2 member ports
+          name: Port-Channel5
+          mode: passive
+          member_interfaces:
+            - name: Ethernet1/4
+              peer: peerDevice2
+              peer_interface: Ethernet1/12
+              speed: "forced 10000full"
+            - # peer, peer_interface not set, use one from parent L3 Port-Channel
```

Comment:
We should not use the parent peer interface in descriptions since it would be misleading (port-channel something). But I agree that we can inherit the peer.

Reply:
Fixed the comment in yml file and underlying logic to not derive `peer_interface` info from parent Port-Channel for a member eth port. 
On a related note, what should we populate for "peer_type" for such member Eth ports under Port-Channel when generating structured config via _get_l3_port_channel_member_ports_cfg() ? 
Currently, choosing not to set any value. Please let me know if otherwise.

Reply:
For now, populating `peer_type` as `l3_port_channel_member` for a member eth intf under `l3_port_channel`
(being used for documentation purposes only)